### ISSUE TITLE
Recurring Events with a duration do not appear properly on week and day view

### ISSUE THREAD

**Issue by cheng1994** [fullcalendar/fullcalendar#4635]
There's an issue with recurring events using rrule showing up on day view and week view with duration. The events appear properly on month view but when looking at week view a multi day event that runs into the next week doesn't show properly. In regards to day view the event only appears for the start date and every day after doesn't appear.

https://jsfiddle.net/gordawn/oxftLq6d/16/

**Comment by acerix** [comment]
Thanks for the report and demo, I can see the issue and also that it works as expected for simple events (not using rrule)

https://jsfiddle.net/hg6Lztkx/ 

Related to this issue:

https://github.com/fullcalendar/fullcalendar/issues/4617

**Comment by cheng1994** [comment]
@acerix Is this planned for the next minor release?

**Comment by arshaw** [comment]
I see what's going on. I'll have this fixed for the next release.

**Comment by arshaw** [comment]
fixed in 4.2.0

https://github.com/fullcalendar/fullcalendar/releases/tag/v4.2.0

**Comment by alex-osman** [comment]
I'm having this problem with 4.2.0

JSFiddle: https://jsfiddle.net/a0qx2udf/10/

```
events: [{
    title: 'Test',
    daysOfWeek: ['1', '2', '3'],
    startTime: "08:30",
    endTime: "00:30",
}]
```
